### DISCUSSION TITLE
Datasource issue

### DISCUSSION THREAD

**Discussion by DanGMaher** [code4recovery/12-step-meeting-list#880] [Q&A] [answered]
We are in the process of having the DCMs in the Spokane, WA area update the meeting information on the Spokane Central Office website [aaspokane.org](http://aaspokane.org/) and having the data transferred to [area92aa.org](http://area92aa.org/) site.  We installed the Advanced Access Management plugin on [aaspokane.org](http://aaspokane.org/) to allow the DCMs to manage only their district meetings.  I set up the datasource on [area92aa.org](http://area92aa.org/), and we have duplicate meeting info showing up on [area92aa.org](http://area92aa.org/) for these four districts.  

My plan to resolve this issue was to export a CSV of all the meetings on [area92aa.org](http://area92aa.org/), delete the four district meetings and import the CSV.  

I stopped because of these issues.
1. I know the user management uses the "Editor" of the meeting info, and CSV doesn't show any editor info. (there is an author column with no data)  If I delete existing meeting info, I am concerned that the editor info will be lost.
2. I am not sure which import setting to use.
       **When importing...
don't delete anything
delete only the meetings, locations, and groups for the regions present in this CSV
delete all meetings, locations, groups, districts, and regions
delete all meetings, locations, and groups not from a data source
l would appreciate any help I can get.**
thanks,

--
Dan Maher
Technology Chair for WSEA92.
509-551-6750

**Comment by kiyote33** [comment, marked as answer]
Hi @DanGMaher, I think you pretty much have the correct plan to remove duplicates from your site:

1. **Export the current site** to a meetings.CSV file using the Export Meeting List feature from our Import page.
2. Remove [aaspokane.org meetings](https://aaspokane.org/meetings) from a copy of the meetings.csv file which will be imported via a data source feed.
3. I would suggest selecting "delete all meetings, locations, groups, districts, and regions" after choosing your modified copy of the meetings.csv file.
4. Hit the **Begin** button to Import your CSV file.
5. When the CSV Import is finished, **Refresh** the Spokane data source feed to add those meetings and regions to your listing.

As regards to your concern about losing data used by another plugin, I've not used our 12 Step Meeting List (TSML) plugin in conjunction with the "Advanced Access Manager" plugin, but I would expect all of your data related to TSML to be preserved in your original meetings.csv export.
Hope this helps alleviate your concerns.
